gpkit/_mosek/cli_expopt.py

```diff
@@ -36,16 +36,16 @@ def imize_fn(path=None, clearfiles=True):
         path = tempfile.mkdtemp()
     filename = path + os.sep + "gpkit_mosek"
     if "mosek_bin_dir" in settings:
-        os.environ['PATH'] = ":".join([os.environ['PATH'],
-                                       settings["mosek_bin_dir"]])
+        if settings["mosek_bin_dir"] not in os.environ["PATH"]:
+            os.environ["PATH"] = (os.environ["PATH"]
+                                  + ":" + settings["mosek_bin_dir"])
```

Comment:
good fix... 

`:.join(` seemed cleaner to me than +, just curious, was it intentional to switch?

Reply:
Yeah, I think it's less confusing to use "+" when it's always a two-element list.